.github/workflows/build-sr2023.yml

```diff
@@ -0,0 +1,33 @@
+name: Build SR2023 floorplan outputs
+
+on:
+  push:
+    paths:
+      - '2023/**'
+      - '.github/workflows/build-sr2023.yml'
+jobs:
+  build-2023:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r 2023/requirements.txt
+
+      - name: Build sr2023 floorplans
+        run: |
+          cd 2023
+          ./generate_svg.py
```

Comment:
```suggestion
          ./generate_svg.py --teams team_names.txt layouts/
```

Reply:
Why?

Reply:
(those are already the defaults)

Reply:
Team names are not templated by default and `specs` is a required argument.

Reply:
> Team names are not templated by default

Was there a reason why not?

Given that the `team_names.txt` (which I renamed to `team_names.yaml` as it was actually a YAML file) is committed to the repo and had useful content, I'd changed it to just be picked up by default (https://github.com/srobo/comp-floorplans/commit/59e96c194e3e33bd8c069c7c6e266285a0e040ce). Frankly the lack of picking it up seemed like a bug.
Happy to change the default back if that's an issue though. (I had initially expected that the `generate_svg.py` script also munged the `map.svg`, though it doesn't).

> and `specs` is a required argument.

Again, why? Since this had been assigned a default value I had assumed that the lack of picking that up was a bug and fixed it in https://github.com/srobo/comp-floorplans/commit/37c2e098e1bcb312570672f3899e51a684b2cce2.

Reply:
https://github.com/srobo/comp-floorplans/pull/22/commits/0ec02c6121abfdafdc9e5d6b7f1fd501877549ee flips back the default for the teams file.